Kava/Kava/Source/PageObjects/Infrastructure/Block.swift

```diff
@@ -113,11 +113,11 @@ public final class ApplicationBlock : Block {
 /**
  A concrete block that represents a "page", usually correlating to a view controller
 */
-public class PageBlock : Block {
+open class PageBlock : Block {
     
-    public private(set) var session: TestSession
-    public var backingElement: XCUIElement = XCUIApplication()
-    public var parentBlock: Block?
+    open fileprivate(set) var session: TestSession
```

Comment:
Change back to `public private(set)`. Shouldn't be overridden, and it shouldn't be modified externally